src/Native/ElmFire.js

```diff
@@ -82,7 +82,13 @@ var _ThomasWeiser$elmfire$Native_ElmFire = function () {
       var rest = locationList._1;
       switch (head._0) {
         case 'url':
-          ref = new Firebase (head._1);
+          if (!firebase.apps.length) {
+            firebase.initializeApp({databaseURL: head._1});
+          }
+          ref = firebase.database();
+          break;
+        case 'ref':
+          ref = getRefStep (rest) .ref (head._1);
```

Comment:
That's my style for JS code.
I would appreciate if contributors would stick the style of the existing code.